Attempts are made by the default Hurd init script to start services that are
not yet in the package collection.  Skip that stuff.

--- daemons/rc.sh
+++ daemons/rc.sh
@@ -94,6 +94,8 @@ if test -d /var/run; then
 fi
 echo done
 
+exit 0
+
 # This file must exist for e2fsck to work. XXX
 touch /var/run/mtab
 
